scheduler/src/common/helper/garms.helper.ts

```diff
@@ -196,6 +212,125 @@ export const createGarmsCashFileDetails = (
   }
 };
 
+export const createGarmsCreditFile = (
+  transactions: Transaction[],
+  garmsExtractType: GarmsExtractType,
+  permitServiceCodes: Map<string, number>,
+  logger: Logger,
+) => {
+  const fileName = 'GARMS_CREDIT' + Date.now();
```

Comment:
No space or hyphen between the constant and date? Also, will timezone be an issue?